shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -35,35 +63,94 @@
     private static final float SCROLL_POSITION_CAP_FOR_INFINITY = 70000.0f;
     private static final int ROOT_NODE_ID = 0;
 
-    private final FlutterView owner;
+    /// Value is derived from ACTION_TYPE_MASK in AccessibilityNodeInfo.java
+    private static int FIRST_RESOURCE_ID = 267386881;
+
+    // Real Android View, which internally holds a Flutter UI.
+    @NonNull
+    private final View rootAccessibilityView;
+
+    // Direct interface between Flutter's Android embedding and the Flutter framework.
+    @NonNull
+    private final FlutterJNI flutterJNI;
+
+    // The accessibility communication API between Flutter's Android embedding and
+    // the Flutter framework.
+    @NonNull
     private final AccessibilityChannel accessibilityChannel;
+
+    // The top-level Android View within the containing Window.
+    // TODO(mattcarroll): why do we need this? and why are we caching it?
+    @NonNull
     private final View decorView;
-    private Map<Integer, SemanticsObject> objects;
-    private Map<Integer, CustomAccessibilityAction> customAccessibilityActions;
+
+    // TODO(mattcarroll): what exactly is a semantic object, and why do we cache them?
+    @NonNull
+    private final Map<Integer, SemanticsObject> semanticsObjects;
+
+    // TODO(mattcarroll): what makes a CustomAccessibilityAction different than a regular
+    //                    AccessibilityAction. why do we cache them?
+    @NonNull
+    private final Map<Integer, CustomAccessibilityAction> customAccessibilityActions;
+
+    // TODO(mattcarroll): what is the source of truth for this supposed to be? Is it
+    //                    whether or not the OS has accessibility enabled? Or do other
+    //                    factors influence this value?
     private boolean accessibilityEnabled = false;
-    private SemanticsObject a11yFocusedObject;
+
+    // The widget within Flutter that currently has the focus of Android's
+    // accessibility system.
+    @Nullable
+    private SemanticsObject accessibilityFocusedObject;
+
+    // The widget within Flutter that currently has the focus of Android's input
+    // system.
+    // TODO(mattcarroll): is there a relationship between this and the InputMethodManager?
```

Comment:
The difference is one corresponds to a text field and other to a semantics node. Their representation might be different